standalone/src/main/java/io/pravega/local/InProcPravegaCluster.java

```diff
@@ -239,8 +244,8 @@ private void startLocalSegmentStores() throws Exception {
      */
     private void startLocalSegmentStore(int segmentStoreId) throws Exception {
         Properties authProps = new Properties();
-        authProps.setProperty("pravega.client.auth.method", "Default");
-        authProps.setProperty("pravega.client.auth.userName", "arvind");
+        authProps.setProperty("pravega.client.auth.method", "Pravega-Default");
+        authProps.setProperty("pravega.client.auth.userName", "admin");
```

Comment:
The `InProcPravegaCluster` builder takes a username and password, shouldn't they be used here in lieu of hardcoded defaults?  Meanwhile, in standalone mode the values should come from `standalone-config.properties`.

Reply:
This change is unrelated to this PR. Please file another PR to fix this, or explain how this is leading to the issue being fixed.